conf/messages

```diff
@@ -1057,6 +1057,8 @@ itemQuantity.error.required = Enter an amount
 itemQuantity.error.nonNumeric = Amount must be in numbers
 itemQuantity.error.range = Amount can be up to 15 digits long, with up to 3 decimal places
 itemQuantity.error.decimalPlaces = Amount can have up to 3 decimal places
+itemQuantity.error.mustNotBeZero.noShippingMarksExist = The packaging quantity cannot be 0 because there are no shipping marks for any of the packages in this movement
+itemQuantity.error.mustNotBeZero.atLeastOnePackageWithQuantityMoreThanZero = The packaging quantity cannot be 0 as there must be at least one package with the same shipping mark that has a quantity greater than 0
```

Comment:
The copy-deck has slightly different content `The packaging quantity cannot be 0 as there must be at least one package with this shipping mark that has a quantity greater than 0.`